src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -103,7 +103,27 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig {
     tezosConf.network
   )
 
-  try {mainLoop(0)} finally {shutdown()}
+  /**
+    * Tries to fetch blocks head to verify if connection with Tezos node was successfully established
+    */
+  @tailrec
+  def checkConnection(): Unit = {
+    Try {
+      Await.result(tezosNodeOperator.getBlockHead(tezosConf.network), lorreConf.connectionCheckTimeout)
+    } match {
+      case Failure(e) =>
+        logger.error("Could not connect to the Tezos node", e)
+        Thread.sleep(lorreConf.retryInterval.toMillis)
+        checkConnection()
+      case Success(_) =>
+        logger.info("Successfully connected to the Tezos node")
```

Comment:
How about "Successfully made initial connection to Tezos"?

Reply:
changed according to suggestion